google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -84,6 +85,20 @@
 
     /** Builds the backup from this builder. */
     public abstract Backup build();
+
+    /**
+     * Output Only.
+     *
+     * <p>Returns the max allowed expiration time of the backup, with microseconds granularity.
+     */
+    protected abstract Builder setMaxExpireTime(Timestamp maxExpireTime);
```

Comment:
This class is public, so you will have to remove the abstract and provide a default implementation. Otherwise, this would be a breaking change (which requires a major release).